src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -9,8 +9,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>478</width>
-    <height>553</height>
+    <width>609</width>
```

Comment:
Why the increase in width?

Reply:
I don't know; this dropped out of the ui designer. I changed it back. Surprisingly it does not makle the slightest differrence what number I put there. The dialog always has identical dimensions.